app/search/template.hbs

```diff
@@ -92,16 +106,121 @@ as |layout|>
                 {{/each}}
             </ul>
         {{/if}}
-        {{#each this.filterableProperties as |filterableProperty|}}
-            {{#let filterableProperty.indexCard as |propertyCard|}}
-                <Search::-Components::FilterFacet
-                    @cardSearchText={{this.q}}
-                    @cardSearchFilters={{this.activeFilters}}
-                    @propertyCard={{propertyCard}}
-                    @propertySearch={{filterableProperty}}
-                    @toggleFilter={{this.toggleFilter}}
-                />
-            {{/let}}
+        {{#each this.filterableProperties as |filterableProperty index|}}
+            {{#if (eq index 1)}}
+                {{#let filterableProperty.indexCard as |propertyCard|}}
+                    <div id='fundersFilter'>
```

Comment:
Sure.

Reply:
I remember testing this and have just re-confirmed with testing. It will fail without the global namespace for the id being exposed to the popover element. We've discussed unique-id, but this is currently not functional as mentioned during scrum. If you find another way, that would be great but right now this is the solution I found.